Makefile

```diff
@@ -1,24 +1,23 @@
 .PHONY: requirements
 
 requirements: requirements.js
-	pip install -q -r requirements/base.txt --exists-action w
+	pip install .
 
 requirements.js:
 	npm install
 
-requirements.test: requirements
-	pip install -q -r requirements/test.txt --exists-action w
-
 install: requirements
 
-develop: requirements.test
+develop: requirements
+	pip install -U pytest>=2.7 pytest-mock pycodestyle pylint edx-lint==0.5.1
 
 clean:
 	find . -name '*.pyc' -delete
```

Comment:
We should have some easy convention for deleting the crawler results. I think we had a longer discussion on this elsewhere in the PR so I'll comment over there.